03-attribute-operations.Rmd

```diff
@@ -342,21 +325,20 @@ wb_north_america
 ```
 
 In this book, we focus on spatial data. 
-Most of the following examples will have a `sf` object as the first argument and a `data.frame` object as the second argument. 
-A new `sf` object will be a result of these joins. 
-However, the reverse order is also possible and will result in a `data.frame` object.
+Most of the following join examples will have a `sf` object as the first argument and a `data.frame` object as the second argument which results in a new `sf` object.
+However, the reverse order is also possible and will give you back a `data.frame` object.
 This is mostly beyond the scope of this book, but we encourage you to try it.
 
 ### Left joins
 
```

Comment:
One could think about just presenting the two most important join-types such as the inner- and left join (supported by `st_join`, and leave the rest as an excercise to the reader mentioning again the join chapter in the R for data science book.